app/Providers/JetstreamServiceProvider.php

```diff
@@ -29,6 +30,14 @@ public function register()
      */
     public function boot()
     {
+        Fortify::registerView(function () {
+            return view('vue.auth.register');
```

Comment:
Why are we creating `vue` folder? You can overwrite the existing `auth.register` with new code and same for login view.